rocky/OOI_database_seed.json

```diff
@@ -502,13 +502,26 @@
     "fields": {
       "last_updated": "2021-08-09T14:50:09.835Z",
       "data": {
-        "description": "The firewall may be configured in a riskful manner.",
-        "recommendation": "Manually validate whether this open ports should actually be open.",
+        "description": "An uncommon open port was found to be open. A firewall may be configured in a riskful manner.",
+        "recommendation": "Manually validate whether this open port should actually be open.",
         "risk": "Middle"
       },
       "consult_api": false
     }
   },
+  {
+    "model": "tools.ooiinformation",
+    "pk": "KATFindingType|KAT-OPEN-COMMON-TCP-PORT",
+    "fields": {
+      "last_updated": "2021-08-09T14:50:09.835Z",
+      "data": {
+        "description": "A port commonly used for TCP traffic was found to be open.",
+        "recommendation": "Manually validate whether this open port should be open.",
+        "risk": "Recommendation"
```

Comment:
I don't think we should make findings for this. Having a finding for every webserver that has port 80 and 443 open isn't useful. As @underdarknl said in the linked ticket, we want to have policy objects for this. Until we have those we shouldn't create lots of useless findings in my opinion.

Reply:
Not every application hosting a webserver is actually expected by the system administrator to host a webserver. In addition, not every application hosting a webserver should also be reachable by a KAT installation (maybe you only intennd your webserver to be reachable in your internal network). In both these cases it is a relevant finding if port 80 and/or 443 are reachable by KAT.
In addition, adding a finding for ports makes sure any port find by KAT wil also end up in a report (export). A clean table with IPs and found open ports would be a nicer but more complicated solution in this case though.

Reply:
Would a filtered tree view (ip's + their ports) exported as PDF / CSV not make more sense to include in the report bundle?
I agree some common ports might be unexpected on some machines, but that hinges on knowledge we don't have in the graph at the moment. Creating findings from those ports regardless just to include them in a finding report creates more objects than required, and diminishes the value of the concept Finding.